### PR TITLE
Relationship

### PR DESCRIPTION
**Pull request by zero060699**
Screenshot from 2020-12-28 21-48-09
Screenshot from 2020-12-28 21-48-25
Screenshot from 2020-12-28 21-49-21

### CODE REVIEW

**Comment by zero060699** [comment]
ready

File: database/migrations/2020_12_22_013941_create_posts_table.php [hunk 1]
```diff
@@ -14,10 +14,11 @@ class CreatePostsTable extends Migration
     public function up()
     {
         Schema::create('posts', function (Blueprint $table) {
-            $table->id();
+            $table->bigIncrements('id');
             $table->string('name');
             $table->string('detail');
             $table->string('author');
+            $table->integer('user_id');
```
Comment (huyenttm-0901) [inline database/migrations/2020_12_22_013941_create_posts_table.php]: ko sửa file migrate (luom)

**Comment by zero060699** [comment]
ready

File: app/Http/Controllers/PostController.php [hunk 2]
```diff
@@ -74,10 +76,10 @@ public function update(PostRequest $request, $id)
         $posts = Post::findOrFail($id);
         $posts->update($request->all());
         if ($posts) {
-            return redirect()->route('posts.index')->with('success', trans('success'));
+            return redirect()->route('posts.index')->with('success', trans('label.success'));
         }
 
-            return redirect()->route('posts.edit', $id)->with('fail', trans('error'));
+            return redirect()->route('posts.edit', $id)->with('fail', trans('label.error'));
```
Comment (congnx-2423) [inline app/Http/Controllers/PostController.php]: convention

File: app/Http/Controllers/PostController.php [hunk 3]
```diff
@@ -42,12 +43,13 @@ public function store(PostRequest $request)
...
             'author' => $request->author,
+            'user_id' => Auth::id(),
         ]);
         if ($posts) {
-            return redirect()->route('posts.index')->with('success', trans('successfully'));
+            return redirect()->route('posts.index')->with('success', trans('label.successfully'));
         }
 
-            return redirect()->route('posts.index')->with('fail', trans('error'));
+            return redirect()->route('posts.index')->with('fail', trans('label.error'));
```
Comment (congnx-2423) [inline app/Http/Controllers/PostController.php]: convention

**Comment by zero060699** [comment]
ready

File: app/Http/Controllers/PostController.php [hunk 4]
```diff
@@ -42,12 +43,12 @@ public function store(PostRequest $request)
...
             'author' => $request->author,
+            'user_id' => Auth::id(),
         ]);
         if ($posts) {
-            return redirect()->route('posts.index')->with('success', trans('successfully'));
+            return redirect()->route('posts.index')->with('success', trans('label.successfully'));
         }
-
-            return redirect()->route('posts.index')->with('fail', trans('error'));
+            return redirect()->route('posts.index')->with('fail', trans('label.error'));
```
Comment (congnx-2423) [inline app/Http/Controllers/PostController.php]:
```suggestion
        return redirect()->route('posts.index')->with('fail', trans('label.error'));
```

File: app/Http/Controllers/PostController.php [hunk 5]
```diff
@@ -74,10 +75,9 @@ public function update(PostRequest $request, $id)
         $posts = Post::findOrFail($id);
         $posts->update($request->all());
         if ($posts) {
-            return redirect()->route('posts.index')->with('success', trans('success'));
+            return redirect()->route('posts.index')->with('success', trans('label.success'));
         }
-
-            return redirect()->route('posts.edit', $id)->with('fail', trans('error'));
+            return redirect()->route('posts.edit', $id)->with('fail', trans('label.error'));
```
Comment (congnx-2423) [inline app/Http/Controllers/PostController.php]:
```suggestion
        return redirect()->route('posts.edit', $id)->with('fail', trans('label.error'));
```

**Comment by zero060699** [comment]
ready

**Review by congnx-2423 — APPROVED** [review]

File: app/Http/Controllers/PostController.php [hunk 6]
```diff
@@ -42,12 +43,12 @@ public function store(PostRequest $request)
...
             'author' => $request->author,
+            'user_id' => Auth::id(),
         ]);
         if ($posts) {
-            return redirect()->route('posts.index')->with('success', trans('successfully'));
+            return redirect()->route('posts.index')->with('success', trans('label.successfully'));
         }
-
-            return redirect()->route('posts.index')->with('fail', trans('error'));
+        return redirect()->route('posts.index')->with('fail', trans('label.error'));
```
Comment (huyenttm-0901) [inline app/Http/Controllers/PostController.php]: 1 dòng trống trc return

**Pull request merged**